pimd/pim_ssm.c

```diff
@@ -84,6 +85,13 @@ static int pim_is_grp_standard_ssm(struct prefix *group)
 
 	return prefix_match(&group_ssm, group);
 }
+#else
+static int pim_is_grp_standard_ssm(struct prefix *group)
+{
+	return (group->u.prefix6.s6_addr32[0] & htonl(0xFF300000)) ==
```

Comment:
wrong mask here (should be `& htonl(0xFFF00000)`)

and also need to check prefix length

Reply:
Resolved